rubin_sim/maf/maf_contrib/selfcal_uniformity_metric.py

```diff
@@ -55,11 +62,10 @@ def __init__(self, nside_residual=128, highglat_cut=30.0, outlier_nsig=4.0):
         self.outlier_nsig = outlier_nsig
 
         self.units = "mmag"
+        self.filter_name = filter_name
 
     def run(self, data_slice, slice_point=None):
-        filter_name = data_slice["filter"][0]
```

Comment:
Well, that's interesting and I think I hadn't noticed. I don't think Eli had specified the constraint on the data either ... 
So, ha. Definitely should recheck the more downtime / less downtime outputs. 
I also wonder how well the metric tracks against the actual photometric calculation that Eli was expecting to do given that FGCM works across all filters (I think).